packages/ui/src/components/TimelineItem/TimelineItem.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+import { Meta, Story } from '@storybook/react';
+
+import { TimelineItem, TimelineItemProps } from './TimelineItem';
+
+const meta: Meta = {
+  title: 'Components / TimelineItem',
+  component: TimelineItem,
+};
+
+export default meta;
+
+const Template: Story<TimelineItemProps> = (args) => (
+  <Box bg="gray.200" p="40px">
```

Comment:
💭 On the website it would be displayed on a white background so I think you can use the same bg color here